app/models/rbac.rb

```diff
@@ -304,9 +304,8 @@ def self.find_targets_without_rbac(_klass, scope, find_options = {})
   end
 
   def self.minimum_columns_for(klass)
-    columns = ['id']
-    columns << 'type' if klass.column_names.include?('type') # STI classes will instantiate calling class without type column
-    columns.join(', ')
+    # STI classes will instantiate calling class without type column
+    klass.column_names.include?('type') ? %w(id type) : %w(id)
```

Comment:
These are passed into `select`, so I split them out.
I'll revert to the old form